### PR TITLE
feat(data): Two more person ships: Intola and Verdorve

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Content (Person Ships)**

## Summary

Follow-up to #136 and #135. Re-PR of #143 after a git mistake.
Intola is a Scout with two Heavy Lasers as armament. I wondered about giving it a Quantum Keystone, but that's a little spoilery.
Verdorve's captain loves plasma weapons and engines, and the Splinter Verdorve flies uses nothing but plasma.

Halves the person spawn rate.

### CODE REVIEW

File: data/gamerules.txt [hunk 1]
```diff
@@ -24,9 +24,9 @@ gamerules
 	"universal ramscoop" false
 	
 	# An attempt to spawn a person ship happens on average every this many frames.
-	# DEFAULT: 72000 frames (twenty minutes)
+	# DEFAULT: 9000 frames (two and a half minutes)
```
Comment (Zitchas) [inline data/gamerules.txt]:
```suggestion
	# DEFAULT: 36000 frames (ten minutes)
```

I like the sentiment, but I think boosting the person spawn rate shorter than ten minutes is premature at this point. Probably another twenty people or so before we can start looking at making it shorter than ten.

File: data/gamerules.txt [hunk 2]
```diff
@@ -24,9 +24,9 @@ gamerules
...
 	# ALLOWABLE VALUES: any integer >= 1
-	"person spawn period" 72000
+	"person spawn period" 9000
```
Comment (Zitchas) [inline data/gamerules.txt]:
```suggestion
	"person spawn period" 36000
```

**Review by Zitchas — CHANGES_REQUESTED** [review]
Overall I like them. They have some fun dialog.

I do think it is s little pre mature to boost the spawn rate that far, though I definitely agree with bringing it back to ten from twenty...

**Comment by TheGiraffe3** [comment]
I kind of want to make a new "Anti-Pirate" government for these two.
They don't care a bit about any of the human factions, but want to shoot pirates on sight.

**Review by Zitchas — APPROVED** [review]
Back in one of the EV games there were two merchant governments. The default merchant ran in the face of danger, consistently. They avoid *any* combat to the best of their ability. Then there was the "Brave merchant" that acts like our current one, which would seek out enemies and the like and only run away once they got damaged a bunch. Kind of missing that behavior. It always seems weird to me that so many merchants will charge into combat.

For now, though, I think it looks good.

**Comment by TheGiraffe3** [comment]
Alright, sounds good.
I think if we use personalities correctly we could get that behavior, but yeah, not as part of a government... yet.

**Pull request merged**